projects/fsastorefrontlib/src/cms-components/dynamic-forms/form-components/dynamic-select/dynamic-select.component.ts

```diff
@@ -33,10 +37,27 @@ export class DynamicSelectComponent extends AbstractFormComponent
   ngOnInit() {
     super.ngOnInit();
     if (this.config.apiValue) {
+      this.configureApiValueForCategory();
       this.setFormControlValuesFromAPI();
     }
   }
 
+  configureApiValueForCategory() {
+    if (!this.config.apiValue.url.includes(CATEGORY_CODE + '=')) {
```

Comment:
can we drop this + '=' part?
Condition will still be fullfiled right? And it looks better